docs/unattend.xml

```diff
@@ -0,0 +1,79 @@
+<!--A sample unattend.xml which helps in setting admin password and running scripts on first boot -->
```

Comment:
Is this a valid JSON file? I ran this through https://jsonformatter.curiousconcept.com/ to format it better and it returned an error.

Reply:
This is just plain xml file

Reply:
I think I should stop reviewing :-) Please run this through https://www.freeformatter.com/xml-formatter.html or something similar and post a more readable version.